onnxruntime/core/framework/execution_frame.cc

```diff
@@ -329,7 +329,7 @@ Status ExecutionFrame::AllocateMLValueTensorSelfOwnBufferHelper(OrtValue& ort_va
   if (static_cast<uint64_t>(len) > std::numeric_limits<size_t>::max()) {
     return Status(ONNXRUNTIME, INVALID_ARGUMENT, "Tensor shape is too large");
   }
-  if (!IAllocator::CalcMemSizeForArrayWithAlignment<64>(static_cast<size_t>(len), element_type->Size(), &size)) {
+  if (!IAllocator::CalcMemSizeForArrayWithAlignment<256>(static_cast<size_t>(len), element_type->Size(), &size)) {
```

Comment:
How about defining a constant variable for 256 in some header file and then refer it here and in file session_state.cc, tensorprotoutils.cc, utils.cc below?

Reply:
Why are we settling for 256 and not 64? I understand 256 is CUDA preferred but this can also lead to increased memory usage. In either case we should measure memory footprint across workloads to measure the impact.